changelog/0.39.md

```diff
@@ -81,3 +81,19 @@ Released 2024-07-15
 ### Other(s)
 
 - [#2187](https://github.com/elastisys/compliantkubernetes-apps/pull/2187) - bug: Fix bug preventing configuration of PSA level @simonklb
+
+## v0.39.2
+
+Released 2024-09-02
+
+## Release highlights
+
+- Drops support for Kubernetes v1.25, adds support for Kubernetes v1.30.<br>Nginx version upgrade to `1.25.5`.
```

Comment:
This feels pretty major for doing in a patch release, no?

Reply:
Yes, but it is required to patch the vulnerability. And 1.25 has been EOL for quite some time, as 1.28 is the current last version. And we only support the current versions.

Reply:
Good rational! Rather break outdated clusters than being attacked through this exploit. :smile: :+1: